vtds_cluster_kvm/private/config/config.yaml

```diff
@@ -140,19 +140,6 @@ cluster:
       # The name of the Blade Interconnect network underlying this
       # Virtual Network. Set this to null for blade-local networks.
       blade_interconnect: base-interconnect
```

Comment:
REVIEWERS: here is where I remove the second source of truth from the configuration. A block like this one will now be computed and put into the deployment configuration sent to the blades, but will not need to be in any manually written configuration.